mobile_app_connector/models/request.py

```diff
@@ -58,17 +65,15 @@ def mobile_contact_us(self, parameters, **params):
             'name': question,
             'categ_id': self.env['crm.claim.category'].sudo().search(
                 [('name', '=', source)]).id,
-            'partner_id': contact_id,
-            'user_id': False
+            'partner_id': contact_id or partner.id,
+            'user_id': False,
+            'language': self.sudo().detect_lang(question).lang_id.code
         })
         claim.message_post(body=question,
                            subject=_("Original request from %s %s ") %
                            (firstname, lastname))
-
-        if contact_id:
-            partner = self.env['res.partner'].browse(int(contact_id))
-            if partner.exists():
-                self.create_email_for_interaction_resume(subject, question, partner)
```

Comment:
This should not removed, as the route can send directly the id of the logged partner! We won't ignore that and replace by an email search which could not find the partner.